app/bundles/CoreBundle/Assets/css/app/less/components/navs.less

```diff
@@ -171,7 +171,7 @@
 .nav-tabs-wrapper:has(.nav-tabs-contained) + form > .tab-content,
 .nav-tabs-contained + .tab-content,
 .nav-tabs-contained + form > .tab-content {
-  background-color: var(--layer);
+  // background-color: var(--layer);
 }
```

Comment:
Can this be removed when it's not needed?

Reply:
I'll need the selectors for the PR that solves permanently :)

Reply:
this is a quick-fix to allow releasing the next version, because the permanent solution involves recreating all pages that have tabs to add wrappers (I'll do it using a template then)

Reply:
I'll bring the snippet to a Google Doc and remove from the codebase

Reply:
done btw